docs/cli-commands.md

```diff
@@ -1,6 +1,17 @@
 # CLI commands
 
 
+Command  | Description
+---------| -----------------------------------------------------------------
+`create [name]` | Create a new aea project.
+`scaffold connection/protocol/skill [name]`  | Scaffold a new connection, protocol, or skill.
+`add connection/protocol/skill [name]`  | Add connection, protocol, or skill to agent.
+`remove connection/protocol/skill [name]` | Remove connection, protocol, or skill from agent.
```

Comment:
Consider adding an article.
"Add connection, protocol, or skill to an/the agent"
"Remove connection, protocol, or skill from an/the agent"